portfolio/src/main/webapp/index.html

```diff
@@ -5,21 +5,35 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+    body {
+      background-image: url('https://i.pinimg.com/originals/92/17/4a/92174ad3758fdb8f9c2e4579ad5430a0.jpg');
+      background-size: cover;
+    }
+    </style> -->
   </head>
   <body>
+    <div class="header" style="background-image: url('https://wallpaperaccess.com/full/1425559.jpg'); background-size: cover;">
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns:

Separate structure from presentation from behavior.

Strictly keep structure (markup), presentation (styling), and behavior (scripting) apart, and try to keep the interaction between the three to an absolute minimum.